azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/robots/HomeScreenRobot.kt

```diff
@@ -56,7 +56,7 @@ class HomeScreenRobot : ScreenRobot<HomeScreenRobot>() {
     }
 
     fun clickAlertDialogOkButton() {
-        waitUntilViewIdIsDisplayed(android.R.id.button1)
+        waitUntilViewAndTextIsDisplayed(android.R.id.button1, "OK")
```

Comment:
Would you consider naming this waitUntilTextOnViewIsDisplayed(....)? The current one is fine too.

Reply:
done